src/email/logs.contoller.js

```diff
@@ -26,6 +30,7 @@
             $scope.logs = rows;
             $scope.totalItems = EmailLogsService.totalRecords;
             $scope.currentPage = EmailLogsService.currentPage;
+            $scope.isLoading = false;
```

Comment:
I think if you are going to make motech-list be event driven by a 'loading' event, you don't need the variable $scope.isLoading, correct?